ansible/roles/tileserver_step2/templates/settings.js

```diff
@@ -7,7 +7,7 @@ var maxZoom = 19; // maximum zoom level the tile server offers
 // define both base maps
 var baseLayers = {
     'local' : L.tileLayer('http://localhost/tiles/osm/{z}/{x}/{y}.png', {maxZoom: maxZoom, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors'}),
-    'osm.org' : L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {maxZoom: 19, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, imagery CC-BY-SA'})
+    'osm.org' : L.tileLayer('https://.tile.openstreetmap.org/{z}/{x}/{y}.png', {maxZoom: 19, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, imagery CC-BY-SA'})
```

Comment:
there's a left-over `.` here. While fixing it, it's also worth fixing the attribution by linking to `https://www.openstreetmap.org/copyright`